src/test/java/com/googlecode/lanterna/gui2/ComboBoxTest.java

```diff
@@ -15,7 +15,7 @@ public static void main(String[] args) throws IOException, InterruptedException
 
     @Override
     public void init(WindowBasedTextGUI textGUI) {
-        final BasicWindow window = new BasicWindow("ComboBoxTest");
+        final BasicWindow window = new BasicWindow();
```

Comment:
Were these two changes intended, or did they slip in?

It seems to me as if some features are being *removed* here...
 src/test/java/com/googlecode/lanterna/gui2/ComboBoxTest.java  and src/test/java/com/googlecode/lanterna/gui2/ListBoxTest.java